scan/src/ApolloClient.re

```diff
@@ -0,0 +1,13 @@
+// in memory cache for caching GraphQL data
+let graphQL = "d3n-debug.bandprotocol.com:5433/v1/graphql";
```

Comment:
Please make this configurable. It will be different depending on whether we run on https.